src/_guides/libraries/c-interop.md

```diff
@@ -168,6 +168,41 @@ For details, see the following:
 * [Flutter dart:ffi page][binding]
 * [dart:ffi examples]({{page.samples}})
 
+## Interfacing with native types
+
+The dart:ffi library provides multiple types 
+which represent native types in C.
+
+They all implement [`NativeType`][] and
+serve purely as markers in type signatures.
```

Comment:
This is not true for `Pointer`, `Array` and user-defined subtypes of `Union` and `Struct`.

Reply:
Maybe split the table in two down below? The ones that are truly only markers, and the others.